js/mail.js

```diff
@@ -627,6 +627,17 @@ var Mail = {
 					data.replyCc = draft.cc;
 					data.replyBcc = draft.bcc;
 					data.replyBody = draft.body;
+				} else {
+					if(data.hasHtmlBody) {
+						/*// Doesn't work here, because html of mail is rendered later
+						// Needs info! also
+						// TODO language support
+						jQuery.get(data.htmlBodyUrl, function(text) {
+							data.replyBody = '> ' + jQuery(text).text().replace(/\n/g, '\n> ');
+						});*/
+					} else {
+						data.replyBody = '\n\nOn ' + data.date + ' ' + data.from + ' wrote:\n> ' + jQuery(data.body).text().replace(/\n/g, '\n> ');
```

Comment:
We need to make this translatable. But I’m actually thinking if it’s better in a simpler format. Instead of:
* On 17 mai 2015 14:40:18 UTC+2 Jan-Christoph Borchardt wrote:
Better
* Jan-Christoph Borchardt (17 May 2015, 14:40)

Like a chat format.